File: TransCoder_tokenized_test_set_functions/cpp/FIND_ELEMENTS_ARRAY_LEAST_TWO_GREATER_ELEMENTS_2.cpp
void findElements ( int arr [ ], int n ) {
  int first = INT_MIN, second = INT_MIN;
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( arr [ i ] > first ) {
      second = first;
      first = arr [ i ];
    }
    else if ( arr [ i ] > second ) second = arr [ i ];
  }
  for ( int i = 0;
  i < n;
  i ++ ) if ( arr [ i ] < second ) cout << arr [ i ] << " ";
}

File: TransCoder_tokenized_test_set_functions/python/FIND_ELEMENTS_ARRAY_LEAST_TWO_GREATER_ELEMENTS_2.py
def findElements ( arr, n ) :
    first = - sys . maxsize
    second = - sys . maxsize
    for i in range ( 0, n ) :
        if ( arr [ i ] > first ) :
            second = first
            first = arr [ i ]
        elif ( arr [ i ] > second ) :
            second = arr [ i ]
    for i in range ( 0, n ) :
        if ( arr [ i ] < second ) :
            print ( arr [ i ], end = " " )
    
